test/functional/ex_cmds/cd_spec.lua

```diff
@@ -280,6 +279,10 @@ describe("getcwd()", function ()
   end)
 
   it("returns empty string if working directory does not exist", function()
+    if helpers.iswin() then
+      pending('[Cannot delete working directory in Windows]')
```

Comment:
Can't ever fix this on Windows AFAIK, so we should silently `return` instead of marking it `pending()`.